tests/test_subparser.py

```diff
@@ -0,0 +1,263 @@
+from dataclasses import dataclass, field
+from typing import Union
+from sys import version_info
+
+from pytest import mark, raises
+
+from pydargs import parse
+
+
+@dataclass
+class Action1:
```

Comment:
Would you still consider renaming Action to SubCommand or Command? I think that's the terminology used in CLI world

Reply:
Agreed, done ✅